CondeNast/subscription-forms
# New Yorker should use the mobile markup instead of the responsive

This:
- does not make any changes to the original source code found here: https://subscribe.newyorker.com/subscribe/newyorker/118423
- is updating `subscribe/newyorker/118423.html` to use the mobile markup found here when using a mobile device: https://subscribe.newyorker.com/subscribe/newyorker/118423
- is prettifying `subscribe/newyorker/118423.html`
- is adding a dev script to be able to make changes locally, while accessing the assets hosted on `https://subscribe.newyorker.com` using this dev script: `npm run dev:subscribe.newyorker.com`

### Original markup:
![screen shot 2018-12-05 at 12 39 05 pm](https://user-images.githubusercontent.com/5905197/49532590-dfe06300-f88a-11e8-9e7b-9f211f5070fc.png)

### Local markup:
![screen shot 2018-12-05 at 12 39 16 pm](https://user-images.githubusercontent.com/5905197/49532600-e40c8080-f88a-11e8-9b2f-680f753f12bc.png)

To test:
1. `npm i && npm run dev:subscribe.newyorker.com`
2. Open two tabs in chrome using the smallest device in your responsive devices in chrome, for example: `iPhone 5/SE`
3. Open these two pages in the tabs:
- https://subscribe.newyorker.com/subscribe/newyorker/118423
- http://localhost:8000/subscribe/newyorker/118423

4. They should look identical. (There maybe some script errors that are not being handled by this PR)
